src/tests/utils.py

```diff
@@ -187,12 +187,13 @@ def remove_all_competition_data(self):
 
         # tasks and solutions
         for task in Task.objects.all():
-            created_files += [
-                task.ingestion_program.data_file.name,
-                task.scoring_program.data_file.name,
-                task.input_data.data_file.name,
-                task.reference_data.data_file.name,
-            ]
+            if competition.bundle_dataset is not None:
```

Comment:
Is this what we want to do here? Does this clean up 100% of the run? If you run selenium tests there may be files left over from the run, we want to stop that